fili-core/src/main/java/com/yahoo/bard/webservice/config/BardFeatureFlag.java

```diff
@@ -7,8 +7,9 @@
  */
 public enum BardFeatureFlag implements FeatureFlag {
     PARTIAL_DATA("partial_data_enabled"),
-    DRUID_CACHE("druid_cache_enabled"),
+    @Deprecated DRUID_CACHE("druid_cache_enabled"),
     DRUID_CACHE_V2("druid_cache_v2_enabled"),
```

Comment:
@QubitPi  collapse the cache config parameter into one called `druid_query_response_caching_strategy`, this parameter takes a string of one of the following: `none`, `v2`. `etag`. Put comment on the config parameter.